CONTRIBUTING.md

```diff
@@ -203,13 +203,18 @@ Open a pull request against the main `opentelemetry-dotnet` repo.
 * If the PR is not ready for review, please mark it as
   [`draft`](https://github.blog/2019-02-14-introducing-draft-pull-requests/).
 * Make sure CLA is signed and all required CI checks are clear.
-* Submit small, focused PRs addressing a single
-  concern/issue.
+* Submit small, focused PRs addressing a single concern/issue.
 * Make sure the PR title reflects the contribution.
 * Write a summary that helps understand the change.
 * Include usage examples in the summary, where applicable.
 * Include benchmarks (before/after) in the summary, for contributions that are
   performance enhancements.
+* We are open to bot generated PRs or AI/LLM assisted PRs. Actually, we are
+  using
+  [dependabot](https://docs.github.com/en/code-security/dependabot/dependabot-security-updates/configuring-dependabot-security-updates)
+  to automate the security updates. However, if you use bots to generate spams,
```

Comment:
This language `if you use bots to generate spams,` seems a little odd.

Probably at the least "spams" should be "spam PRs" or "spammy PRs"? Maybe a bit of definition would also help?

`if you use bots to generate spam PRs (eg incorrect, noisy, non-improvements, unintelligible, etc.),`

Reply:
Good point, I've tweaked the wording.